hyperon_das_atomdb/adapters/ram_only.py

```diff
@@ -538,10 +538,9 @@ def delete_atom(self, handle: str, **kwargs) -> None:
     def create_field_index(
         self,
         atom_type: str,
-        field: str,
+        fields: List[str],
         type: Optional[str] = None,
```

Comment:
`type` is a builtin function in python, so, ideally, it should not be used to name vars/params.

Reply:
fixed